org.opent2t.sample.lamp.superpopular/com.smartthings.lightbulb/js/thingTranslator.js

```diff
@@ -207,6 +207,7 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
 
         dim.id = 'dim';
         dim.dimmingSetting = providerSchema['attributes'].level;
+        if (dim.dimmingSetting > 100 ) dim.dimmingSetting = 100;
```

Comment:
Thanks, will try it out with hardware.

---
In reply to: [112694398](https://github.com/openT2T/translators/pull/235#discussion_r112694398) [](ancestors = 112694398,112323616)